pkg/kubelet/dockertools/manager.go

```diff
@@ -64,6 +64,8 @@ const (
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
 	ndotsDNSOption = "options ndots:5\n"
+
+	containerNetworkEnvKey = "NETWORK"
```

Comment:
+1 to what @thockin wrote. We don't force particular env var names onto users.

See: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/downward-api.md

cc @pmorie